modules/wkt/src/lib/encode-wkt.ts

```diff
@@ -1,23 +1,25 @@
 // Fork of https://github.com/mapbox/wellknown under ISC license (MIT/BSD-2-clause equivalent)
-/* eslint-disable prefer-template */
 
 /**
  * Stringifies a GeoJSON object into WKT
- */
-
-/**
  * @param geojson
  * @returns string
  */
 
 export default function encodeWKT(gj: {
   type: string;
   properties?: {};
-  geometry?: any;
+  geometry?: {
+    type: string;
+    coordinates: any[];
+  };
```

Comment:
Should we use `@types/geojson` typing for `geometry` and `geometries`?

Reply:
So the input type can be `gj: Feature | Geometry` where both are imported from `@types/geojson`